src/main/java/net/liquidpineapple/pang/objects/Ball.java

```diff
@@ -0,0 +1,123 @@
+package net.liquidpineapple.pang.objects;
+
+import java.util.Random;
+
+/**
+ * Created by Erik on 7-9-2016.
+ */
+public class Ball extends GameObject {
+
+    public enum Directions{
+        LEFT, RIGHT, RANDOM
+    }
+
+    //Size 1 is the smallest ball.
+    int size;
+    int movX;
+    int movY;
+    private static final String textureLocationBlue = "/sprites/Balls/Blue/frame 1.png";
+    private static final String textureLocationGreen = "/sprites/Balls/Green/frame 1.png";
+    private static final String textureLocationRed = "/sprites/Balls/Red/frame 1.png";
+    private static final String textureLocationYellow = "/sprites/Balls/Yellow/frame 1.png";
+    /**
+     * Creates a ball with a set horizontal speed and a variable vertical speed depending on size.
+     *
+     * @param startX    X-coordinate to make the ball
+     * @param startY   Y-coordinate to make the ball
+     * @param direction Direction the ball should move, valid inputs are: "left" and "right", anything else randomizes the direction.
+     * @param sizeIn    The desired size of the ball, Greater then 4 or smaller then 0 results in a size 4 ball.
+     */
+    public Ball(int startX, int startY, Directions direction, int sizeIn){
+        super(textureLocationBlue, startX, startY);
+        if(size == 1){
+            ChangeImage(textureLocationGreen);
+        }
+        else if(size == 2){
+            ChangeImage(textureLocationRed);
+        }
+        else if (size == 3){
+            ChangeImage(textureLocationYellow);
+        }
+
+        if(sizeIn>0 && sizeIn<5){
+        size = sizeIn;}
+        else {
+            size = 4;}
+
+        //Makes the ball go either left or right depending on input, if the input is incorrect it randomly chooses a direction.
+        if(direction.equals(Directions.LEFT)){
+            movX=-15;}
+        else if(direction.equals(Directions.RIGHT)){
```

Comment:
@Wolgo Please adhere to the styling conventions here. The `} else` not being on a separate line drastically reduces readability